FV3/gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -2722,6 +2723,7 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     logical              :: norad_precip      = .false.      !< radiation precip flag for Ferrier/Moorthi
     logical              :: lwhtr             = .true.       !< flag to output lw heating rate (Radtend%lwhc)
     logical              :: swhtr             = .true.       !< flag to output sw heating rate (Radtend%swhc)
+    logical              :: do_clearsky_rad   = .false.      !< flag for whether to do only clear-sky radiation
```

Comment:
This is a flag for 'whether' to compute clear-sky radiation, or 'only'?  Since you zero out the cloud data, that does sound like 'only.' It is well documented and makes sense, just want to double check I am interpreting correctly.

Reply:
Yeah, clear-sky radiation is computed regardless of what this flag is set to. But if this flag is set to True, then the "total-sky" radiation is equal to the clear-sky radiation, and the heating rate/radiative fluxes seen by other components of the model are the clear-sky.

I agree the name could be better. I'm gonna go with @spencerkclark's suggestion of `do_only_clearsky_rad` if you agree!